vcpkg-init/mint-standalone-bundle.ps1

```diff
@@ -72,16 +81,23 @@ try {
     }
 
     $bundleConfig = @{
-        'readonly' = $False;
+        'readonly' = $readonly.isPresent;
         'usegitregistry' = $True;
         'embeddedsha' = $sha
     }
+    New-Item -Path "out/.vcpkg-root" -ItemType "File"
 
     Set-Content -Path "out/vcpkg-bundle.json" `
         -Value (ConvertTo-Json -InputObject $bundleConfig) `
         -Encoding Ascii
-
-    tar czf $DestinationTarballName -C out *
+    if ($DestinationDir) {
+        if (-not (Test-Path $DestinationDir)) {
+            mkdir $DestinationDir
+        }

```

Comment:
```suggestion
        New-Item -Path $DestinationDir -ItemType 'Directory' -Force
```

Reply:
In my branch I changed this to just `Move-Item $TempDir $DestinationDir`